bundle/API/Repository/Values/Content/Query/SortClause/ContentTypeId.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Netgen\TagsBundle\API\Repository\Values\Content\Query\SortClause;
+
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
+
+/**
+ * Sets sort direction on Content Type ID for a content query.
+ */
+class ContentTypeId extends SortClause
```

Comment:
Would content type identifier be a better sort clause than the ID? It makes more sense to sort by the identifier than a numeric database ID, right?

Reply:
Yes it would, but identifier is not being indexed in Solr.

Reply:
Really? That's unfortunate...